fabric-resource-loader-v0/build.gradle

```diff
@@ -1,2 +1,2 @@
 archivesBaseName = "fabric-resource-loader-v0"
-version = getSubprojectVersion(project, "0.2.2")
+version = getSubprojectVersion(project, "0.3.0")
```

Comment:
this is just a bug fix. a major increment is not needed

Reply:
That isn’t a major increment
and it’s adding new features without breaking existing ones
hence minor increment

Reply:
a bug fix is the 3rd number not the second.

Reply:
doshy please read what i've written
> and it’s adding new features without breaking existing ones
hence minor increment

while this fixes a bug that's not its main purpose